.changeset/gentle-chefs-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': major
```

Comment:
```suggestion
'@shopify/polaris': patch
```

Potentially controversial but I think this is more of a patch change? Since it's fixing a bug but not actually a breaking change to behaviour. Maybe minor at most?